src/components/index/IndexComponent.tsx

```diff
@@ -21,13 +79,85 @@ const Index = () => {
           <HeaderTitleText> Requests &amp; Proposals </HeaderTitleText>{" "}
         </HeaderTitle>
       </Header>
+      <FilterWrapper>
+        <FilterButtonRow>
+          <FilterButton
+            variant={filter === Filter.DEFAULT ? "primary" : "outline"}
+            onClick={() => setFilter(Filter.DEFAULT)}
+          >
+            <div>All</div>{" "}
+            <FilterNumbers selected={filter === Filter.DEFAULT}>
+              {numAll}
+            </FilterNumbers>
+          </FilterButton>
+          <FilterButton
+            variant={filter === Filter.REQUESTS ? "primary" : "outline"}
+            onClick={() => setFilter(Filter.REQUESTS)}
+          >
+            <div>Requests </div>{" "}
+            <FilterNumbers selected={filter === Filter.REQUESTS}>
+              {numRequested}
+            </FilterNumbers>
+          </FilterButton>
+          <FilterButton
+            variant={filter === Filter.PROPOSED ? "primary" : "outline"}
+            onClick={() => setFilter(Filter.PROPOSED)}
+          >
+            <div>Proposed </div>{" "}
+            <FilterNumbers selected={filter === Filter.PROPOSED}>
+              {numProposed}
+            </FilterNumbers>
+          </FilterButton>
+          <FilterButton
+            variant={filter === Filter.DISPUTED ? "primary" : "outline"}
+            onClick={() => setFilter(Filter.DISPUTED)}
+          >
+            <div>Disputed </div>{" "}
+            <FilterNumbers selected={filter === Filter.DISPUTED}>
+              {numDisputed}
+            </FilterNumbers>
+          </FilterButton>
+          <FilterButton variant="base">
+            <input
```

Comment:
this input box is really hard to press, it might be better to allow the whole button to check/uncheck

Reply:
Fixed :)